packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,29 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
 
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-center() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
 
+@mixin table-skeleton($skeleton-height) {
```

Comment:
again, this mixin is not table specific but would be useful as a way to customize the appearance of a skeleton.

however, how useful are these styles? given that only `height` changes, do we really need to set the other three properties every time or can we set them once on `.pt-skeleton` and not have a mixin?